appeals/web/src/server/appeals/appeal-details/issue-decision/issue-decision.controller.js

```diff
@@ -111,30 +115,25 @@ export const renderIssueDecision = async (request, response) => {
 /**
  *
  * @param {import("express-session").Session & Partial<import("express-session").SessionData>} session
- * @param {string} documentType
+ * @param {string} decisionType
  */
-function storeFileUploadInfo(session, documentType) {
-	const { inspectorDecision = {} } = session;
-	if (!inspectorDecision.fileUploadInfo) {
-		inspectorDecision.fileUploadInfo = {};
-	}
-
-	if (session.fileUploadInfo) {
-		inspectorDecision.fileUploadInfo[documentType] = cloneDeep(session.fileUploadInfo);
-		session.inspectorDecision = inspectorDecision;
-	}
+function storeFileUploadInfo(session, decisionType) {
+	const { outcome } = session[decisionType] || {};
+	session[decisionType] = cloneDeep({ outcome, ...session.fileUploadInfo });
+	delete session.fileUploadInfo;
 }
 
 /**
  *
  * @param {import("express-session").Session & Partial<import("express-session").SessionData>} session
- * @param {string} documentType
+ * @param {string} decisionType
  */
-function restoreFileUploadInfo(session, documentType) {
-	const { fileUploadInfo = {} } = session.inspectorDecision || {};
+function restoreFileUploadInfo(session, decisionType) {
+	// eslint-disable-next-line no-unused-vars
```

Comment:
You might be able to remove this if you can delete the `outcome` variable, it doesn't seem to be used